### PR TITLE
Fixing redis locking for 5000 seconds unintentionally

### PR DESCRIPTION
**Pull request by karljin**
Same problem as what we see in transmissions

### CODE REVIEW

File: umeboshi/utils.py [hunk 1]
```diff
@@ -15,13 +15,13 @@
 
 
 @contextlib.contextmanager
-def simple_lock(lock_key, timeout=5000):
```
Comment (manav) [inline umeboshi/utils.py:18]: is 5 sec enough ? should we make it longer.

**Comment by markhsu-ms** [comment]
@karljin I think this PR is still valid right?

**Review by manav — CHANGES_REQUESTED** [review]
I don't recall anymore. Would recommend a fresh look. I would keep the number of unknowns minimum for py3 migrations.

File: umeboshi/utils.py [hunk 2]
```diff
@@ -15,13 +15,13 @@
...
+def simple_lock(lock_key, timeout=5000, redis_timeout_sec=5):
```
Comment (markhsu-ms) [inline umeboshi/utils.py:18]: `timeout` is defined but never used after the change. Are we doing it intentionally?

File: umeboshi/utils.py (hunk 1 above)
Comment (karljin) [reply umeboshi/utils.py:18]: it's always been meant to be 5 seconds. the fact that it's locking for 5000 seconds is purely accidental and "bad"

File: umeboshi/utils.py (hunk 2 above)
Comment (karljin) [reply umeboshi/utils.py:18]: yes. if i remember correctly, there's a very similar code in site that does use "timeout" somewhere so I wanted this to be very explicit (that it's a redit lock and it's in seconds, whereas "timeout" is in ms)

**Review by manav — APPROVED** [review]
assuming this can be tested properly

File: umeboshi/utils.py (hunk 2 above)
Comment (markhsu-ms) [reply umeboshi/utils.py:18]: Oh sorry it's used here 

```
if waited > timeout:
```

Conversation resolved.

**Review by markhsu-ms — APPROVED** [review]
LGTM

**Pull request merged**